### DISCUSSION TITLE
Different return result for internal HTTP API requests

### DISCUSSION THREAD

**Discussion by vvitkovsky** [bluenviron/mediamtx#1933] [General]
Hi gents, 

I have one proposal related internal HTTP API - currently mediamtx has a bit different behavior for similar logic, e.g.: 

/v2/webrtcsessions/get/{name} returns HTTP 200 (OK) with empty response if no WebRTC session with specified name was not found,
/v2/paths/get/{name} returns HTTP 404 (Not found) if such path was not found.

For my opinion it will be good if both will return HTTP 404 (Not found) because this logically more clear for a request for a non-existing resource (same for other get requests for HLS, RTSP, RTMP etc).

Best regards, 
Victor

**Comment by aler9** [comment]
i've created an issue here: #1994
